elastic/kibana
# [ResponseOps][Cases] Allow users to select template in the case action

## Summary

Resolves https://github.com/elastic/kibana/issues/187203

This PR allows users to select cases template while creating case action. It will help users to create cases with specific templates for a rule. 

<img width="620" alt="Screenshot 2024-08-30 at 13 17 37" src="https://github.com/user-attachments/assets/9a9225f2-3af4-4922-a935-fae3cfe0cb8e">


### How to verify

- Create a template in `cases > settings`
- Create a rule
- Add case action and select the template
- Save rule
- Verify the case generated by case action is as per template
- Also verify case generated by case action without any template works as before.

### Checklist

### For maintainers



### Release notes:
Allow users to select template while adding a case action in the rule.
